### PR TITLE
Add Tribes promo graphic to town square

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- [X] Add Tribes promo graphic to town square.
- [X] Display for all users
- [X] User should be able to dismiss this, once they've seen it.

##### Refers/Fixes
Add Tribes promo graphic to town square #6217

##### Testing
image

### CODE REVIEW

File: app/assets/v2/images/tribes/logo-townsquare.svg [hunk 1]
```diff
@@ -0,0 +1,424 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (octavioamu) [inline app/assets/v2/images/tribes/logo-townsquare.svg]: can we optimize this with https://jakearchibald.github.io/svgomg/ ? As a recommendation don't use inkscape they create svg attr and tags out of standards only works with  inkscape, as a free solution I recommend using figma (their export is very easy also) 
But in general I always use https://jakearchibald.github.io/svgomg/ at the end.

File: app/assets/v2/images/tribes/promo-decoration.svg [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```
Comment (octavioamu) [inline app/assets/v2/images/tribes/promo-decoration.svg]: same https://jakearchibald.github.io/svgomg/

**Comment by zoek1** [comment]
done @octavioamu

**Comment by PixelantDesign** [comment]
great!

**Pull request merged**